pkgs/applications/misc/vym/default.nix

```diff
@@ -14,15 +14,17 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "vym";
-  version = "2.8.42";
+  version = "2.9.26";
 
   src = fetchFromGitHub {
     owner = "insilmaril";
     repo = "vym";
-    rev = "89f50bcba953c410caf459b0a4bfbd09018010b7"; # not tagged yet (why??)
-    hash = "sha256-xMXvc8gt3nfKWbU+WoS24wCUTGDQRhG0Q9m7yDhY5/w=";
+    rev = "v${finalAttrs.version}";
```

Comment:
:+1: but why with `finalAttrs`?

Reply:
1. I don't like `rec`, by the arguments present here:
https://nix.dev/guides/best-practices.html#recursive-attribute-set-rec

2. Because of it I use the finalAttrs design pattern.

   Rec-less, overlay-style overridable attrset for stdenv.mkDerivation was introduced in https://github.com/NixOS/nixpkgs/pull/119942.

Reply:
Thanks for the link to that PR :eyes: